### PR TITLE
Add latest tag for releases

### PR DESCRIPTION
**Pull request by huyhg**

### CODE REVIEW

File: cloudbuild-tag.yaml [hunk 1]
```diff
@@ -8,23 +8,51 @@ steps:
   args:
   - -exc
   - |
-    docker pull gcr.io/marketplace-k8s-app-tools/k8s/dev:sha_$COMMIT_SHA
-    docker tag gcr.io/marketplace-k8s-app-tools/k8s/dev:sha_$COMMIT_SHA gcr.io/$PROJECT_ID/k8s/dev:$TAG_NAME
+    docker pull gcr.io/$_STAGING_PROJECT_ID/k8s/dev:sha_$COMMIT_SHA
```
Comment (trironkk) [inline cloudbuild-tag.yaml]: Is there a place we can see all of the triggers that will need to be retrofitted?

Maybe that design doc should have a section for it?

File: cloudbuild-tag.yaml [hunk 2]
```diff
@@ -8,23 +8,51 @@ steps:
...
+  args:
+  - -ceu
+  - |
+    if [[ "$(cat should_tag_latest)" == "true" ]]; then
+      docker tag gcr.io/$PROJECT_ID/k8s/dev:$TAG_NAME gcr.io/$PROJECT_ID/k8s/dev:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm/onbuild:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm/onbuild:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller/onbuild:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_helm_tiller/onbuild:latest
+      docker tag gcr.io/$PROJECT_ID/k8s/deployer_envsubst:$TAG_NAME gcr.io/$PROJECT_ID/k8s/deployer_envsubst:latest
```
Comment (trironkk) [inline cloudbuild-tag.yaml]: I have a comment here to discuss with you offline.

Comment (trironkk) [reply cloudbuild-tag.yaml]: Sync'd offline - need to invoke `docker push`.

We can revisit and clean up later.

File: cloudbuild-tag.yaml (hunk 1 above)
Comment (huyhg) [reply cloudbuild-tag.yaml]: This file affects only one trigger, in `cloud-marketplace-tools` prod project, which has already been retrofitted.

**Pull request merged**